Gemfile

```diff
@@ -93,11 +95,12 @@ gem 'citeproc-ruby', '~> 2.0.0'
 gem 'csl-styles', '~> 2.0.0'
 gem 'bibtex-ruby', '~> 5.1.0'
 
-gem 'omniauth', '~> 1.3.1'
-gem 'omniauth-ldap', '~> 1.0.5'
+gem 'omniauth', '~> 2.1.0'
+gem 'gitlab_omniauth-ldap', git: 'https://gitlab.com/gitlab-org/omniauth-ldap.git', ref:'2584e8703418f9608ba35e1c0634515b242c2c35'
```

Comment:
Could be installed through rubygems: https://rubygems.org/gems/gitlab_omniauth-ldap and tied to '~> 2.2.0'